Dockerfile

```diff
@@ -13,7 +13,7 @@ COPY . .
 RUN make test
 
 # Build the binary
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="${LDFLAGS:-}" -o /app/sroperator cmd/main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=${BUILDARCH} go build -ldflags="${LDFLAGS:-}" -o /app/sroperator cmd/main.go
```

Comment:
is this BUILDARCH an env var or a build arg?

Reply:
Yes, please reference this link:https://docs.docker.com/build/building/multi-platform/